docs/topics/mta-cli-args.adoc

```diff
@@ -0,0 +1,248 @@
+|--exportSummary |Flag to instruct {ProductShortName} to generate an `analysisSummary.json` export file in the output directory. The file contains analysis summary information for each application analyzed, including the number of incidents and story points by category, and all of the technology tags associated with the analyzed applications.
+|--exportZipReport |This argument creates a `reports.zip` file in the output folder. The file contains the analysis output, typically, the reports. If requested, it can also contain the CSV export files.
+|--help |Display the {ProductShortName} help message.
+|--immutableAddonDir |Add the specified directory as a custom read-only add-on repository.
+|--includeTags| A space-delimited list of tags to use. When specified, only rules with these tags will be processed. To see the full list of tags, use the `--listTags` argument.
+|--input |A space-delimited list of the path to the file or directory containing one or more applications to be analyzed. This argument is required.
+|--install |Specify add-ons to install. The syntax is `<GROUP_ID>:<ARTIFACT_ID>[:<VERSION>]`. For example, `--install core-addon-x` or `--install org.example.addon:example:1.0.0`.
+|--keepWorkDirs| Flag to instruct {ProductShortName} to not delete temporary working files, such as the graph database and extracted archive files. This is useful for debugging purposes.
+|--legacyReports| Flag to instruct {ProductShortName} to generate the old format reports instead of the new format reports.
+|--list| Flag to list installed add-ons.
+|--listSourceTechnologies| Flag to list all available source technologies.
+|--listTags| Flag to list all available tags.
+|--list-targets| Flag to list all available target technologies.
+|--mavenize| Flag to create a Maven project directory structure based on the structure and content of the application. This creates `pom.xml` files using the appropriate Java EE API and the correct dependencies between project modules. See also the `--mavenizeGroupId` option.
+|--mavenizeGroupId| When used with the `--mavenize` option, all generated `pom.xml` files will use the provided value for their `<groupId>`. If this argument is omitted, {ProductShortName} will attempt to determine an appropriate `<groupId>` based on the application, or will default to `com.mycompany.mavenized`.
+|--online |Flag to allow network access for features that require it. Currently only validating XML schemas against external resources relies on Internet access. Note that this comes with a performance penalty.
+|--output |Specify the path to the directory to output the report information generated by {ProductShortName}.
+|--overwrite |Flag to force delete the existing output directory specified by `--output`. If you do not specify this argument and the `--output` directory exists, you are prompted to choose whether to overwrite the contents.
```

Comment:
For the `--overwrite` option, same comment as https://github.com/windup/windup-documentation/pull/848#discussion_r1483590519